tests/fixtures/invalid-template-attributes/basic/expected/x/component/component.html

```diff
@@ -0,0 +1,5 @@
+<template lwc:render-mode="light">
```

Comment:
This doesn't need a `render-mode`, and also it's inconsistent with the `renderMode` of the corresponding `component.js` (one is light, the other is shadow).